### PR TITLE
🌐 Update Russian translation of `docs/ru/docs/alternatives.md`

### PR DESCRIPTION
**Pull request by s111d**
Completely retranslated and corrected russian page of _alternatives.md_ in docs. Several errors were found and fixed.

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 4877b21c53ed5220912f4f6ab67859e6e9eab9fe at: https://c324ba00.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit f15ed97be9de5a6cdf863e67491c6498bbb473b4 at: https://3df03451.fastapitiangolo.pages.dev

File: docs/ru/docs/history-design-future.md [hunk 1]
```diff
@@ -1,77 +1,79 @@
...
-При сравнении с альтернативами, выбор падает на него, поскольку он лучше подходит для множества вариантов использования.
+Ему отдают предпочтение перед его предшественниками по причине лучшего соответствия многим вариантам использования.
 
-Многие разработчики и команды уже используют **FastAPI** в своих проектах (включая меня и мою команду).
+Значительное число разработчиков и команд уже использует **FastAPI** в своих проектах (включая меня и мою команду).
 
 Но, тем не менее, грядёт добавление ещё многих улучшений и возможностей.
 
-У **FastAPI** великое будущее.
+Перед **FastAPI** лежит славное будущее.
```
Comment (Jamim) [inline docs/ru/docs/history-design-future.md:77]: `лежит славное будущее` sounds weird. No one says it like this, [almost](https://www.google.com/search?q=%22%D0%BB%D0%B5%D0%B6%D0%B8%D1%82+%D1%81%D0%BB%D0%B0%D0%B2%D0%BD%D0%BE%D0%B5+%D0%B1%D1%83%D0%B4%D1%83%D1%89%D0%B5%D0%B5%22).
An English version is `FastAPI has a great future ahead.`, so, in my opinion, it's worth translating it like this:
```suggestion
**FastAPI** впереди ждёт славное будущее.
```

**Review by Jamim — CHANGES_REQUESTED** [review]
Hello @s111d,

Thanks for your effort! To be honest, I didn't read the whole reworked document yet, but one sentence at the end captured my attention, so I'd like to suggest a minor change.

File: docs/ru/docs/history-design-future.md (hunk 1 above)
Comment (s111d) [reply docs/ru/docs/history-design-future.md:77]: 1. Кавычки на языке запросов Google означают точное сочетание.
2. Последний – не авторитет в русском языке.
3. Как и современный "общепринятый" русский.
4. Будущее... лежит... перед... – нормальный литературный оборот, но, ладно, речь не про литературу.
5. Пусть будет просто "ждёт", иначе тавтология как "солнце садилось вниз" (с) Володарский

**Comment by tiangolo** [comment]
📝 Docs preview for commit afe041c1c82ea17b2eb9f393286f88000f81e5ca at: https://525c072a.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 3896574f15da24ed80f9710cf70b7c12c8d440bb at: https://acff2457.fastapitiangolo.pages.dev

File: docs/ru/docs/history-design-future.md (hunk 1 above)
Comment (Jamim) [reply docs/ru/docs/history-design-future.md]: > Кавычки на языке запросов Google означают точное сочетание

That's exactly why I used them.

> Пусть будет просто "ждёт"

👌🏻

Comment (s111d) [reply docs/ru/docs/history-design-future.md]: Если нужно проверить использование это нужно делать в Корпусе Русского Языка. В самом крайнем случае в Яндексе.

**Comment by tiangolo** [comment]
Thanks @s111d! It would be better to separate this PR into multiple PRs, one per file. That way it's easier for others to review/approve.

**Comment by s111d** [comment]
> Thanks @s111d! It would be better to separate this PR into multiple PRs, one per file. That way it's easier for others to review/approve.

Totally agree, my bad. Will split it a little later.

**Comment by tiangolo** [comment]
@s111d, could you please split this into 3 PRs, one for each file? That way it would be easier to review, approve, and merge. :nerd_face:

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**